.vscode/settings.json

```diff
@@ -3,5 +3,8 @@
 	"editor.codeActionsOnSave": {
 		"source.fixAll.eslint": true
 	},
-	"files.eol": "\n"
+	"files.eol": "\n",
```

Comment:
Can you add this file to .gitignore

Reply:
you can run 

```bash 
git rm  -r .vscode --cached
```

and commit those changes, this will most likely cause issues for other developers setups when they pull your branch